### PR TITLE
remove "Add Events" button from right panel in admin v1

### PR DESCRIPTION
**Pull request by oleksii-front**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/712

### CODE REVIEW

File: src/pages/Admin/Admin.tsx
Comment (0xdevalias) [inline src/pages/Admin/Admin.tsx:398]: > As part of removing this right hand side button, we should also assess what elements it relies on, and whether they have any purpose anymore once it’s removed. If not, they should all be removed as well.
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/internal-sparkle-issues/issues/712#issuecomment-839193139_

Comment (0xdevalias) [reply src/pages/Admin/Admin.tsx:398]: Is the `showCreateEventModal` state still used/needed once we remove this button? Will `AdminEventModal` still be opened here by other areas of the code?

It looks like `setShowCreateEventModal`/`showCreateEventModal` are passed into `VenueInfoComponent` from `VenueDetails`.

After removing this button in `VenueInfoComponent`, it seems the only place `showCreateEventModal`/`setShowCreateEventModal` is used is as props to the `AdminEventModal` (`src/pages/Admin/AdminEventModal.tsx`) component. Double check this, but most if not all of the modals in our app are somewhat 'independent' of the structure of where they are rendered (because they 'float on top' of everything else).

Based on that, we can likely move the `AdminEventModal` 'up' out of `VenueInfoComponent` and render it in `VenueDetails` instead. That will also let us remove the now redundant `setShowCreateEventModal`/`showCreateEventModal` props from `VenueInfoComponent`.

It seems like `EventsComponent` (rendered by `VenueDetails`) also appears to include/render `AdminEventModal` within it's JSX. Since `EventsComponent` is only rendered by `VenueDetails` (which after the refactor above will render `AdminEventModal` for us), and there is no reason to have the code for a modal in the JSX more than once (so long as it is rendered once, it will always show when it's controls are used), then we can likely also remove it from `EventsComponent` (along with the props it will make redundant when we do so, if any)

Comment (0xdevalias) [reply src/pages/Admin/Admin.tsx:398]: > As a creator/admin, I have 3 places to add events, and one is infinitely more useful than the other two.
> 
> _Originally posted by @sofisparkle in https://github.com/sparkletown/internal-sparkle-issues/issues/712_

Looking at the code for the 'Add an Event' button that is described as being the more 'infinitely useful' one, it seems to exist in `src/pages/Admin/AdminVenueRoomDetails.tsx`

- https://github.com/sparkletown/sparkle/blob/staging/src/pages/Admin/AdminVenueRoomDetails.tsx#L167-L176

It uses `setShowCreateEventModal` to show this modal, which is passed in as a prop.

`AdminVenueRoomDetails` is rendered in `AdminVenueRoomsList`, which defines the `useState` (`showCreateEventModal`/`setShowCreateEventModal`)

`showCreateEventModal` is then used to show `AdminEventModal`.

---

So at least the 'bad' and the 'good' versions are rendering the same modal (`AdminEventModal`) and we don't have excess code to remove there.

**Review by 0xdevalias — DISMISSED** [review]
Comments/changes.

File: src/pages/Admin/Admin.tsx
Comment (dmitrymarchuk) [reply src/pages/Admin/Admin.tsx:376]: Removed the button from "Events" tab. Also cleaned up some unused code in the AdminV1-related components. Decided not to do heavy refactoring as Admin V3 is on the way.
In https://github.com/sparkletown/sparkle/pull/1382/commits/9c09b3ba5b3f0dbbc30602d12580a97ecab0af8e

File: src/pages/Admin/Admin.tsx
Comment (mike-lvov) [inline src/pages/Admin/Admin.tsx:360]: Where is coming from? I though we already had the delete button in place

**Review by mike-lvov — APPROVED** [review]
LGTM, but don't merge

File: src/pages/Admin/Admin.tsx
Comment (dmitrymarchuk) [reply src/pages/Admin/Admin.tsx:360]: This change comes from merge-commit. I don't know why is it highlighted here.

Comment (mike-lvov) [reply src/pages/Admin/Admin.tsx]: Can you confirm that there are no `Delete space` button?

Comment (dmitrymarchuk) [reply src/pages/Admin/Admin.tsx]: I don't know if it's a merge error or I accidentally did something wrong but after merging recent changes from staging there really were two "Delete space" buttons. And unused code I remove became used again:) So I think somebody was working on that "Delete space" button and I interfered. Anyway, fixed in https://github.com/sparkletown/sparkle/pull/1382/commits/f8ad957794093233f6547c039f4063c27482d813

File: src/components/atoms/UserAvatar/UserAvatar.tsx
Comment (mike-lvov) [inline src/components/atoms/UserAvatar/UserAvatar.tsx:6]: The imports are broken

File: src/components/organisms/EventModal/EventModal.tsx
Comment (mike-lvov) [inline src/components/organisms/EventModal/EventModal.tsx:8]: same everywhere

**Review by mike-lvov — CHANGES_REQUESTED** [review]

**Review by mike-lvov — APPROVED** [review]

**Pull request merged**
